.github/workflows/main.yml

```diff
@@ -326,7 +326,8 @@ jobs:
       - name: Install Python dependencies
         run: |
           python3 -m pip install ruamel.yaml scons numpy cython sphinx==5.1.1 \
-          sphinxcontrib-katex sphinxcontrib-matlabdomain sphinxcontrib-doxylink pint
+          sphinxcontrib-katex sphinxcontrib-matlabdomain sphinxcontrib-doxylink pint \
```

Comment:
We can probably remove `sphinxcontrib-katex` and remove that configuration from `conf.py` because I believe the `pydata-sphinx-theme` includes Mathjax.